handlers/contact-us-api/src/main/scala/com/gu/contact_us_api/models/ContactUsEnvConfig.scala

```diff
@@ -0,0 +1,3 @@
+package com.gu.contact_us_api.models
+
+case class ContactUsEnvConfig(clientID: String, clientSecret: String, username: String, password: String, token: String, authEndpoint: String, reqEndpoint: String)
```

Comment:
There's an implementation detail leaking here.  If you changed from using environment vars to some other implementation it wouldn't affect this config type.  So perhaps:
```suggestion
case class ContactUsConfig(clientID: String, clientSecret: String, username: String, password: String, token: String, authEndpoint: String, reqEndpoint: String)
```